.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
If  I'm not wrong, this changes affect a VS Code extension that not all of us use, so I wouldn't include it here until the team agrees on doing it.

Reply:
Correct, old commit. Thanks for the review!